packages/packagers/css/src/CSSPackager.js

```diff
@@ -3,14 +3,14 @@
 import {Packager} from '@parcel/plugin';
 
 export default new Packager({
-  async package({bundle}) {
+  async package({bundle, bundleGraph}) {
     let promises = [];
     bundle.traverseAssets({
       exit: asset => {
         // Figure out which media types this asset was imported with.
         // We only want to import the asset once, so group them all together.
         let media = [];
-        for (let dep of bundle.getIncomingDependencies(asset)) {
+        for (let dep of bundleGraph.getIncomingDependencies(asset)) {
```

Comment:
What's the distinction between methods on the Bundle vs on the BundleGraph? It seems we have both sometimes.

Reply:
Yeah I mentioned this here:

https://github.com/parcel-bundler/parcel/pull/3259/files/4508f831cef9b029955073da656784506de0bee3#r306632980

What do you think of just pushing all of these methods to the bundleGraph only?

Reply:
link didn't work but I'm fine with either way, as long as it's consistent.